modules/considerations-scale-down-data-plane.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: REFERENCE
+[id="considerations-scale-down-data-plane_{context}"]
+= Considerations for scaling down the data plane to zero
+
+Review the following considerations before scaling down data plane to zero.
+
+* Use caution when scaling down the data plane to zero. After scaling down the data plane to zero, the workload from the worker nodes disappears.
+
+* The hosted control plane stays up and running. If necessary, you can scale up the `NodePool` resource.
+
+* After scaling down the data plane to zero, some pods in the control plane stay in the `Pending` status.
+
```

Comment:
I'd say "To prevent Node deletion from getting blocked on draining, you can add the `nodeDrainTimeout` field in the `NodePool` resource.